webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/xml/GitArchiverUtils.scala

```diff
@@ -112,7 +112,7 @@ trait GitArchiverUtils {
       case Right(dir) => dir
       case Left(err) =>
         val msg = s"Error when checking required directories '${file.getPath}' to archive in git: ${err.fullMsg}"
-        GitArchiveLoggerPure.logEffect.error(msg)
+        GitArchiveLogger.logEffect.error(msg)
```

Comment:
this change doesn't compile

Reply:
My sed capacity drop around 23:00 :/

Reply:
(the logEffect part should have been removed)